tests/bsim/bluetooth/mesh/src/test_persistence.c

```diff
@@ -1034,6 +1034,7 @@ static void test_reprovisioning_provisioner(void)
 
 	LOG_INF("Provisioning a remote device...");
 	ASSERT_OK(k_sem_take(&prov_sem, K_SECONDS(40)));
+	k_sleep(K_MSEC(200));
```

Comment:
Please add a comment here about why this is 200 milliseconds and why this is required.
When I add some huge changes like PSA support or another settings backend those change timing significantly such kind no name delays confuse a lot. Why they were added? and why exact this digit?
Sometimes it looks like an attempt to relax test conditions to hide issues or bugs that raise from time to time. It is very difficult to maintain such kinds of tests.

Reply:
Will do 👍